src/bot/parse/parsePullRequestBotCommandLine.spec.ts

```diff
@@ -86,14 +86,18 @@ const dataProvider: DataProvider[] = [
     suitName: "bench-bot, no args when not allowed, should return error",
     commandLine: "bot bench",
     expectedResponse: new Error(
-      `Missing arguments for command "bench". Refer to [help docs](http://cmd-bot.docs.com/) and/or [source code](https://github.com/paritytech/command-bot-scripts).`,
+      `Missing arguments for command "bench". Refer to [help docs](http://cmd-bot.docs.com/static/docs/latest.html) and/or [source code](https://github.com/paritytech/command-bot-scripts).`,
```

Comment:
As I mentioned the other day, if you wanna add an interactive way of seeing of seeing the commands, you can use a `details` html tag.

You can do like command bot and add it in all the message so people don't necessary need to press the help button. 

Click below to see how it works:
<details>
  <summary>Available commands</summary>
<br />

  - `command 1`: Does 1 thing
  - `command 2`: Does 2 things
  - `command 3`: Does 3 things

Find all of the documentation [here](http://cmd-bot.docs.com/static/docs/latest.html)

---
</details> 

The code is the following:
```html
 <details>
  <summary>Available commands</summary>
<br />

  - `command 1`: Does 1 thing
  - `command 2`: Does 2 things
  - `command 3`: Does 3 things

Find all of the documentation [here](http://cmd-bot.docs.com/static/docs/latest.html)
</details> 
```

Reply:
Makes sense, although the plans are to avoid chat-bot experience and in future to not have a need in documentation

Reply:
Yeah, I just suggested it as if we already provide all the available commands on the message, it removes a middle step, even for new commands